### PR TITLE
Add sigopt package

### PR DESCRIPTION
**Pull request by pfhayes**

### CODE REVIEW

**Comment by pfhayes** [comment]
Addresses the broken tests

**Comment by pfhayes** [comment]
This is ready to go, anything else needed on my end?

File: recipes/sigopt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  home: http://github.com/sigopt/sigopt-python
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'SigOpt Python API Client'
+
+extra:
+  recipe-maintainers:
+    - pfhayes
+    - alexandraj777
```
Comment (jakirkham) [inline recipes/sigopt/meta.yaml]: Are you ok being a maintainer on this, @alexandraj777?

File: recipes/sigopt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - requests
```
Comment (jakirkham) [inline recipes/sigopt/meta.yaml]: Can drop `requests` from `build` unless it is actually needed at build time.

File: recipes/sigopt/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  run:
+    - python
```
Comment (jakirkham) [inline recipes/sigopt/meta.yaml]: I think you would want `requests` here though. Please make sure the version constraint here matches the one used by the PyPI package.

File: recipes/sigopt/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+test:
+  requires:
+    - requests
```
Comment (jakirkham) [inline recipes/sigopt/meta.yaml]: This should be dropped as `requests` should be a run time dependency.

File: recipes/sigopt/meta.yaml (hunk 1 above)
Comment (alexandraj777) [reply recipes/sigopt/meta.yaml]: Yup, thanks for checking

File: recipes/sigopt/meta.yaml (hunk 2 above)
Comment (pfhayes) [reply recipes/sigopt/meta.yaml]: I just checked and it appears to be necessary at build time

File: recipes/sigopt/meta.yaml (hunk 3 above)
Comment (pfhayes) [reply recipes/sigopt/meta.yaml]: done

File: recipes/sigopt/meta.yaml (hunk 4 above)
Comment (pfhayes) [reply recipes/sigopt/meta.yaml]: done

**Comment by pfhayes** [comment]
Ok, all comments should be addressed now. Thanks for the feedback!

File: recipes/sigopt/meta.yaml (hunk 2 above)
Comment (jakirkham) [reply recipes/sigopt/meta.yaml]: Why is that? Is it getting imported by `setup.py` from somewhere?

Comment (pfhayes) [reply recipes/sigopt/meta.yaml]: That's correct - it's only incidental and could possibly be removed upstream at some point in the future, but at the moment it's required

**Comment by pfhayes** [comment]
Thanks! What are the next steps to get this merged?

**Comment by jakirkham** [comment]
Was just waiting for CI to complete. 😄

**Comment by jakirkham** [comment]
Thanks @pfhayes.

**Pull request merged**